File: 240317/-999/-999.cpp
#include <iostream>
using namespace std;
int main() {
    int Max=-999,Min=999;
    while(true)
    {
        int num;
        cin>>num;
        if(num==-999)break;
        Max=max(Max,num);
        Min=min(Min,num);
    }
    cout<<Max<<" "<<Min;
    return 0;
}

File: 241014/-999/-999.kt
import kotlin.math.max
import kotlin.math.min
fun main()
{
    val lists=readLine()!!.split(" ").map{it.toInt()}
    var maxx=-1000
    var minn=1000
    for(i in 0..99)
    {
        if(lists[i]==-999)break
        maxx=max(maxx,lists[i])
        minn=min(minn,lists[i])
    }
    print("${maxx} ${minn}")
}